### ISSUE TITLE
Color contrast of active tab links does not meet WCAG 2

### ISSUE THREAD

**Issue by ewohnlich** [plone/Products.CMFPlone#3026]
## BUG/PROBLEM REPORT (OR OTHER COMMON ISSUE)

An active/hovered tab in the default barceloneta theme has a background of #7099bf and a white #ffffff font for a color contrast ratio of 3:1. Per https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#larger-scaledef this would only be sufficient for **large** text which has a loose definition of 18pt or 14pt and bold. I don't believe the 17px and 500 font weight definitions here are quite sufficient for that definition.

### What I did:

Contrasted background and foreground color of "tabs" in the globalnav section for active or hovered tabs, which contain a lighter color than non active tabs.

### What I expect to happen:

A color contrast of 4:1 or 3:1 if the font size and weight are sufficient to match the criteria for "large" text.

### What actually happened:

Achieved only a contrast of 3:1 with insufficient font size and/or weight to be considered large.

### What version of Plone/ Addons I am using:
Plone 5.2 out of the box, those this appears to affect Plone 5.x in general

**Comment by jensens** [comment]
This is fixed in Plone 6
